samples/interactive-tutorials/events/import-user-events-big-query.js

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(datasetId) {
+  // [START retail_import_user_events_big_query]
+
+  // Imports the Google Cloud client library.
+  const {UserEventServiceClient} = require('@google-cloud/retail').v2;
+
+  const projectNumber = process.env['GCLOUD_PROJECT'];
+  const projectId = process.env['PROJECT_ID'];
+
+  const dataSchema = 'user_event';
+  const tableId = 'events'; // TO CHECK ERROR HANDLING USE THE TABLE OF INVALID USER EVENTS
+
+  // Placement
+  const parent = `projects/${projectNumber}/locations/global/catalogs/default_catalog`; // TO CHECK ERROR HANDLING PASTE THE INVALID CATALOG NAME HERE
+
+  // The desired input location of the data.
+  const inputConfig = {
+    bigQuerySource: {
+      projectId,
+      datasetId,
+      tableId,
+      dataSchema,
+    },
+  };
+
+  // Instantiates a client.
+  const retailClient = new UserEventServiceClient();
+
+  const IResponseParams = {
+    IImportUserEventsResponse: 0,
+    IImportMetadata: 1,
+    IOperation: 2,
+  };
+
+  const callImportUserEvents = async () => {
```

Comment:
I think slightly easier to read as `async function callImportUserEvents() {`

Reply:
Within all samples we use arrow functions.. This is too much changes

Reply:
The arrow function is a popular features of ES6, it has become a standard as it is more compact and actually convenient to read. If you have any doubts, please create an Issue and we will update the whole code we have submitted with such code style. will it work for you?